### PR TITLE
Feature: Created vi_editor.md to explain in detail.

### PR DESCRIPTION
**Pull request by Deepam14**

### CODE REVIEW

**Review by gitvicky97 — CHANGES_REQUESTED** [review]
I guess this format isn't that good, change format and then pull request

**Review by Deepam14 — COMMENTED** [review]
changed format by using markdown

File: vi_editor.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# **Vi Text Editor**
+
+There are many ways to edit files in Unix. Editing files using the screen-oriented text editor **vi** is one of the best ways. This editor enables you to edit lines in context with other lines in the file.
+
+An improved version of the vi editor which is called the **VIM** has also been made available now. Here, VIM stands for **Vi IM** proved.
+
+vi is generally considered the de facto standard in Unix editors because −
+
+    * It's usually available on all the flavors of Unix system.
```
Comment (gitvicky97) [inline vi_editor.md:9]: The bullet point didn't apply here, please check

File: vi_editor.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  * **Insert mode:**  
+    This mode enables you to insert text into the file. Everything that's typed in this mode is interpreted as input and placed in the file.
+
+vi always starts in the **command mode.** To enter text, you must be in the insert mode for which simply type i. To come out of the insert mode, press the **Esc** key, which will take you back to the command mode.
+
+**Hint** − If you are not sure which mode you are in, press the Esc key twice; this will take you to the command mode. You open a file using the vi editor. Start by typing some characters and then come to the command mode to understand the difference.
+
+# Starting The Vi editor
+
+
```
Comment (gitvicky97) [inline vi_editor.md:34]: Complete the remaining document by adding interactive snap shots of vi editor working on local terminal.

**Pull request merged**